_events/2022/2022-05-dei-diverse-turner.md

```diff
@@ -0,0 +1,68 @@
+---
+title: EDI in the R Community
```

Comment:
Most people don't know what EDI is - I'd write it out fully here.

Reply:
I've flown from there a few times. :smile: Yeah, even those who know may know it as DEI or somesuch instead.